ui/src/app/pages/login/account-creation/account-creation-institution.spec.tsx

```diff
@@ -179,7 +215,7 @@ it('should trigger email check when email is filled in before choosing instituti
   const wrapper = component();
   await waitOneTickAndUpdate(wrapper);
 
-  getEmailInput(wrapper).simulate('change', {target: {value: 'asdf@broadinstitute.org'}});
+  getEmailInput(wrapper).simulate('change', {target: {value: 'institution@broadinstitute.org'}});
```

Comment:
this is a little confusing.  does the original `asdf` work?

Reply:
no since Broad has Restricted agreement

Reply:
I mean: please change the `emailAddresses[]` in the test data so it's clear that they refer to people and not institutions